Doc/source/capabilities.rst

```diff
@@ -0,0 +1,89 @@
+====================
+SpacePy Capabilities
+====================
+
+This page lists some capabilities of SpacePy and, in some cases, of
+other packages that might be of interest to SpacePy users. It is
+organized by topic; searching within this page is recommended. See the
+:ref:`module reference <module_reference>` for every class/function
+available in SpacePy, organized by module.
+
+.. contents::
+   :local:
+
+Array manipulation
+==================
+Various :mod:`~spacepy.toolbox` functions are useful in manipulating
+NumPy arrays. :mod:`~spacepy.datamanager` also contains many functions
+for indexing arrays and manipulating them in ways that do not depend
+on the interpretation of their contents.
+
+Coordinate Transforms
+=====================
+:mod:`~spacepy.coordinates` provides a class for transforming among
+most coordinate systems used in Earth magnetospheric and
+ionospheric physics.
+
+The :ref:`quaternion math <toolbox_quaternions>` functions of
+:mod:`~spacepy.toolbox` support generalized coordinate transforms.
+
+File I/O
+========
+:mod:`~spacepy.pycdf` provides reading and writing of NASA CDF files,
+with additional functionality for those with ISTP-compliant metadata.
+
+:mod:`~spacepy.datamodel` provides easy reading and writing of HDF5
+files. It also supports reading and writing ASCII-based data files
```

Comment:
Should this also mention NetCDF? `fromNC3` wraps the scipy NetCDF3 stuff to bring things into datamodel, and `fromHDF5` reads _most_ NetCDF4.

Reply:
Sure, why not.